scipy/stats/tests/test_multivariate.py

```diff
@@ -818,6 +818,95 @@ def test_fit_correctness(self, dim):
         assert_allclose(mean_est, mean_ref, atol=1e-15)
         assert_allclose(cov_est, cov_ref, rtol=1e-15)
 
+    def test_fit_both_parameters_fixed(self):
+        data = np.ones((2, 1))
+        mean_fixed = 1.
+        cov_fixed = np.atleast_2d(1.)
+        mean, cov = multivariate_normal.fit(data, fix_mean=mean_fixed,
+                                            fix_cov=cov_fixed)
+        assert_allclose(mean, mean_fixed)
+        assert_allclose(cov, cov_fixed)
```

Comment:
As written, I think this test would pass even if `fix_mean` and `fix_cov` were ignored and the fit were computed normally. How about:

```suggestion
        data = np.full((2, 1), 3)
        mean_fixed = 1.
        cov_fixed = np.atleast_2d(1.)
        mean, cov = multivariate_normal.fit(data, fix_mean=mean_fixed,
                                            fix_cov=cov_fixed)
        assert_equal(mean, mean_fixed)
        assert_equal(cov, cov_fixed)
```